ecs/firehose.conf

```diff
@@ -0,0 +1,24 @@
+[SERVICE]
+    Grace 30
+
+[INPUT]
+    Name forward
+    unix_path /var/run/fluent.sock
+
+[FILTER]
+    Name aws
+    Match *
+    ec2_instance_id true
```

Comment:
Sorry... I just realized we shouldn't add this. Because on Fargate it won't work, there's not instance ID there. (Also, this plugin uses the EC2 metadata service, which many ECS customers disable, so its not safe to assume it will always be there). So we can't have it in a built-in generic config for ECS.

Reply:
Can two different configs be an option? For EC2 and Fargate?

Reply:
Since customers will be able to specify multiple configs, we could have a separate config with just the EC2 filter. 

Then if you are running on EC2, you take your base config with the input and output, and you also add the ec2 filter config.

Reply:
Thats also nice.